app/scripts/app-init.js

```diff
@@ -111,14 +99,13 @@ function importAllScripts() {
   }
 }
 
-// eslint-disable-next-line no-undef
-self.addEventListener('install', importAllScripts);
+importAllScripts();
 
 /*
  * Message event listener below loads script if they are no longer available.
  * chrome below needs to be replaced by cross-browser object,
  * but there is issue in importing webextension-polyfill into service worker.
  * chrome does seems to work in at-least all chromium based browsers
  */
-// eslint-disable-next-line no-undef
-chrome.runtime.onMessage.addListener(importAllScripts);
+// eslint-disable-next-line
+chrome.runtime.onMessage.addListener(() => {});
```

Comment:
After the latest changes to `importAllScripts`, can we remove this line and the comments above?